patterns/2-structured/discover-your-innersource.md

```diff
@@ -43,23 +26,26 @@ People don't bother looking for internally developed solutions - they might not
+* Consider using existing tools:
+  * GitHub Enterprise and npmjs have built-in searches on meta-data
+  * PayPal's Agora project (planned to be open sourced) addresses similar challenges with data source integration
 * Implement process change to first check for internal solutions for this problem.
 * Tool with a central view (but people are more inclined to google externally than look internally)
 * Concierge service (guide) to help product people find stuff. Might not scale but could be helpful in the beginning.
-* Need some very visible lighthouse projects that start using inner source components and make positive statements about the inner source program.
-* Establish a common, asynchronous communication channel (e.g., like slack or metamorph or yammer) across team boundaries. This might not scale beyond a certain organization size. It is possible people will start splitting this one channel into multiple channels by topic once traffic gets too high. Note: having one channel for many users of one tool might be considered an anti-pattern because they can't find it unless they already know about it.
+* Need some very visible lighthouse projects that start using InnerSource components and make positive statements about the InnerSource program.
+* Establish a common, asynchronous communication platform (e.g., chat based) across team boundaries.
+  * This might not scale beyond a certain organization size.
+  * When using common chat systems, start with one single chat channel for the InnerSource project, only split into multiple channels by topic once traffic gets too high. This way the channel feels alive. In addition the number of input channels to watch is kept at a minimum for the host team.
+  * Note: having one channel for many users of one tool might be considered an anti-pattern because they can't find it unless they already know about it.
 * Encourage (and reward) owners of reusable code to use the same search engine to continually search for products that are candidates for use and adoption of the reusable code but not currently doing so.
-* Consider creating a marketplace for marketing InnerSource programs (management can use this mechanism to know which InnerSource projects to fund, but seeing how the marketplace reacts).
-
-## Known Instances
-
-TBD
+* Consider creating a marketplace for marketing InnerSource programs (management can use this mechanism to know which InnerSource projects to fund, but seeing how the marketplace reacts). - see [Gig Marketplace](../2-structured/gig-marketplace.md)
```

Comment:
```suggestion
* Consider creating a marketplace for marketing InnerSource programs (management can use this mechanism to know which InnerSource projects to fund, by seeing how the marketplace reacts). - see [Gig Marketplace](../2-structured/gig-marketplace.md)
```

This sentence only makes sense to me when I make the above change. What do you think @jeffabailey?